xsls/body.xsls

```diff
@@ -18,20 +18,27 @@ X:template body (lang) {
     <body>
 
     <div id="banner">
+        <div id="banner-content"></div>
```

Comment:
I'd be happy to promote the banner to be a permanent part of the page layout and not load it as en external resource. The layout shift it produces on slow links is annoying.

It would also make the hamburger less "risky" from being part of the page loaded after the initial render.

Would you recommend this approach?

Reply:
@TasoOneAsia 
> Would you recommend this approach?

?

Reply:
Works for me 👍

Reply:
Then I might start by submitting a precursor PR that makes the banner part of the site layout "proper". Then the hamburger will always have somewhere safe to live.